viewer/app/src/github-api.js

```diff
@@ -51,7 +51,7 @@ export class GithubApi {
         filename = getFlowResultFilenamePrefix(jsonFile);
       } else {
         filename = getLhrFilenamePrefix({
-          finalUrl: jsonFile.finalUrl,
+          finalDisplayedUrl: jsonFile.finalDisplayedUrl,
```

Comment:
won't this not have `finalDisplayedUrl` for pre-10.0 LHRs?

Reply:
Yeah but the backport in file namer will cover us.

Still, we could just pass `jsonFile` directly right?